src/ensembl/src/content/app/entity-viewer/types/transcript.ts

```diff
@@ -6,7 +6,8 @@ export type Transcript = {
   type: 'Transcript';
   id: string;
   symbol: string;
-  so_term: string; // is there a better name for it?
+  so_term?: string; // is there a better name for it?
+  biotype?: string;
```

Comment:
Can you leave a fixme comment for us to remove one of these fields (likely the `biotype`)?

Reply:
There is already a comment in the #273. So I didn't want to add a comment here that would conflict with it when merging.